e2e/wdio/platform/tests/page-footer.e2e-spec.ts

```diff
@@ -0,0 +1,47 @@
+import { PageFooterPo } from '../pages/page-footer.po';
+import {
+    getElementArrayLength,
+    getText,
+    isElementClickable,
+    scrollIntoView,
+} from '../../driver/wdio';
+import {
+    footerTextArr
+} from '../fixtures/appData/page-footer-contents';
+import { checkElementTextValue } from '../../helper/assertion-helper';
+
+describe('Page Footer test suite:', function() {
+    const pageFooterPage = new PageFooterPo();
+    const {
+        pageFooterClickableLink, pageFooterText  } = pageFooterPage;
+
+    beforeAll(() => {
+        pageFooterPage.open();
+    }, 1);
+    
+        it('verify links are clickable', () => {
+            const linksLength = getElementArrayLength(pageFooterClickableLink);
+            for (let i = 0; i < linksLength; i++) {
+                scrollIntoView(pageFooterClickableLink, i);
+                expect(isElementClickable(pageFooterClickableLink, i)).toBe(true, 'link with index ${i} not clickable');
```

Comment:
error message will not work as you wish, if you want to use template literal - quotes should be ``, not ''